lldb/test/API/lang/swift/hashed_container_storing_nested_generic/main.swift

```diff
@@ -0,0 +1,16 @@
+public class A<T> {
+    public func addValue() {
+        self.dict["key"] = B()
+    }
+    private class B {}
+    private var dict: [String: B] = [:]
+    func test() {
+        let val = dict["key"]!;
```

Comment:
Somewhat interestingly, the without the `!` LLDB gets the type correct even before these changes.

Reply:
Can you elaborate on "LLDB gets the type correct even before these changes"?

Minor note the `;` isn't needed.

Reply:
oh, I think I understand what mean